pkg/config/config_test.go

```diff
@@ -49,6 +67,11 @@ var _ = Describe("Allocation operations", func() {
 
 	})
 
+	It("throws an error when no flat-files are found", func() {
+		_, _, err := GetFlatIPAM(true, &types.IPAMConfig{}, "")
+		Expect(err).To(HaveOccurred())
```

Comment:
please match against the specific error using the `MatchError` matcher.

Reply:
I've created a new error type for this and the tests matches on this specifically (preferred that over prefix/string matching)